src/main/java/io/wispforest/owo/serialization/impl/nbt/NbtDeserializer.java

```diff
@@ -0,0 +1,264 @@
+package io.wispforest.owo.serialization.impl.nbt;
+
+import io.wispforest.owo.serialization.*;
+import net.minecraft.nbt.*;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.*;
+import java.util.function.Supplier;
+
+public class NbtDeserializer implements SelfDescribedDeserializer<NbtElement> {
+
+    protected final Deque<Supplier<NbtElement>> stack = new ArrayDeque<>();
+
+    private boolean unsafe = true;
```

Comment:
This doesn't really make it unsafe, though. All it does is make it lax on format errors

Reply:
I would call such unsafe but I am up for a better name